examples/bankaccount/README.md

```diff
@@ -1,4 +1,294 @@
+
+When finished, the washboard should display the concordance provider:
+
+![Washboard with concordance provider loaded](./docs/images/washboard-with-concordance.png)
+
+[wiki-es]: https://en.wikipedia.org/wiki/Domain-driven_design#Event_sourcing
+
+#### 3.2 Start the `keyvalue` provider
+
+To store state from actors like the `projector`, the [`keyvalue` redis provider][wasmcloud-kv-provider] can be used.
+
+You can start the `keyvalue` provider by it's container image (`wasmcloud.azurecr.io/kvredis:0.19.0`):
+
+![Start redis provider](./docs/videos/start-keyvalue-provider.gif)
+
+[wasmcloud-kv-provider]: https://github.com/wasmCloud/capability-providers/tree/main/kvredis
+
+</details>
+
+<details>
+
+<summary>
+
+### 4. 🏎️ Start the relevant actors
+
+</summary>
+
+Start the `projector`, `process_manager` and `aggregate` actors via the washboard.
+
+For example, to start the `projector` actor:
+
+![Start projector for concordance demo in washboard](./docs/videos/start-projector-for-concordance-demo.gif)
+
+Follow the same process for `process_manager` and `aggregate` actors.
+
+</details>
+
+<details>
+
+<summary>
+
+### 5. 🔗 Link the actors to the providers
+
+</summary>
+
+To enable communication between the actors and providers, we need to [link them][wasmcloud-docs-linkdefs].
+
+The actors and providers should have links as outlined below:
+
+| actor             | provider      | link name | contract ID               | values                                                                                                                                                                                                                                             |
+|-------------------|---------------|-----------|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `projector`       | `concordance` | `default` | `wasmcloud:eventsourcing` | `ROLE=projector,INTEREST=account_created,funds_deposited,funds_withdrawn,wire_funds_reserved,wire_funds_released,NAME=bankaccount_projector`                                                                                                       |
+| `projector`       | `keyvalue`    | `default` | `wasmcloud:keyvalue`      | `URL='redis://0.0.0.0:6379/'`                                                                                                                                                                                                                      |
+| `process_manager` | `concordance` | `default` | `wasmcloud:eventsourcing` | `ROLE=process_manager KEY=wire_transfer_id NAME=interbankxfer INTEREST='{"start":"wire_transfer_requested","advance":["wire_funds_reserved","interbank_transfer_initiated"],"stop":["interbank_transfer_completed","interbank_transfer_failed"]}'` |
```

Comment:
These should be comma-separated for consistency, or the projector link should use spaces, or perhaps both should use newlines